### PR TITLE
Publish docs for vp66

### PR DESCRIPTION
**Pull request by JanPrusinowski**

### CODE REVIEW

File: docs/transparent-validation/protectli/laboratory-assembly-guide.md
Comment (dawidlesiak) [inline docs/transparent-validation/protectli/laboratory-assembly-guide.md:1]: The VP66xx does not have a micro-USB port. Instead, they utilize USB-C and RJ45 ports for console.

Comment (macpijan) [reply docs/transparent-validation/protectli/laboratory-assembly-guide.md:1]: @dawidlesiak I guess this comment was meant to be added a few lines below.

@JanPrusinowski please apply the suggestion

File: docs/transparent-validation/protectli/laboratory-assembly-guide.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Introduction
 
 This document describes platform-specific details for assembling Protectli
-VP2410, VP2420, VP4630/VP4650/VP4670, V1210/V1410/V1610 testing stands.
+VP2410, VP2420, VP4630/VP4650/VP4670, V1210/V1410/V1610, VP66xx testing stands.
```
Comment (macpijan) [inline docs/transparent-validation/protectli/laboratory-assembly-guide.md:6]: Let's use XX in others, as we do for VP66

File: docs/transparent-validation/protectli/laboratory-assembly-guide.md [hunk 2]
```diff
@@ -212,6 +250,29 @@ The method of setting and using serial connection is described in the
...
+
+    > Note, that before using the above-mentioned script, it should be modified and
+    `SONOFF_IP` parameter should be set in accordance with obtained Sonoff IP address.
+
+    To perform basic power operations use the below-described commands:
+
+    1. Turn on the power supply:
+
+        ```bash
+        ./sonoff on
```
Comment (macpijan) [inline docs/transparent-validation/protectli/laboratory-assembly-guide.md:267]: But such script does not exist.

We can simply refer to the OSFV CLI: https://github.com/Dasharo/osfv-scripts/tree/main/osfv_cli - all actions are documented there.

Comment (JanPrusinowski) [reply docs/transparent-validation/protectli/laboratory-assembly-guide.md:267]: I copied it from VP46xx - fixed it for both platforms

File: docs/transparent-validation/protectli/laboratory-assembly-guide.md (hunk 1 above)
Comment (JanPrusinowski) [reply docs/transparent-validation/protectli/laboratory-assembly-guide.md:6]: Changed in: 5084c58dada6befae99d63f2a81cb35ccb4a4962

File: docs/transparent-validation/protectli/laboratory-assembly-guide.md (hunk 2 above)
Comment (JanPrusinowski) [reply docs/transparent-validation/protectli/laboratory-assembly-guide.md]: Changed for vp46xx too in: 5084c58dada6befae99d63f2a81cb35ccb4a4962

File: docs/transparent-validation/protectli/laboratory-assembly-guide.md
Comment (JanPrusinowski) [reply docs/transparent-validation/protectli/laboratory-assembly-guide.md:1]: changed

**Review by macpijan — APPROVED** [review]

**Pull request merged**